3_javascript/javascript1/week3/homework.html/remove.js

```diff
@@ -0,0 +1,16 @@
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  delete names[1];
```

Comment:
Interesting choice to use delete method 🤔  You can read about this here : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete . As you can see , it is better to use for object.
But for this exercise, I would recommend to use splice method : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice which is better when you have data stored in array :)

Reply:
Okay done